content/stratis-release-notes-2.4.1.md

```diff
@@ -0,0 +1,52 @@
++++
+title = "Stratis 2.4.1 Release Notes"
+date = 2021-05-12
+weight = 18
+template = "page.html"
+render = true
++++
+
+*mulhern, Stratis Team*
+
+Stratis 2.4.1 is a bug fix release, which addresses a flaw in the
+multi-threading implementation.
+
+A user could observe the problem by finding that most commands issued by the 
```

Comment:
This wording is slightly confusing to me. Would you be okay with something along the lines of "A user could observe the behavior caused by the bug when CLI commands would either take far longer to complete than normal or the D-Bus connection would eventually time out." I'm not set on this wording, I just want to highlight both cases.